### DISCUSSION TITLE
Closable TabView

### DISCUSSION THREAD

**Discussion by binarrii** [primefaces/community#373] [PrimeVue] [answered]
Closable for `TabView` and/or `TabMenu` tabs like in **PrimeReact** as follow

![image](https://github.com/primefaces/primevue/assets/8007836/8eae424e-9792-401e-b846-fae3b51fb3fe)

**Comment by tugcekucukoglu** [comment, marked as answer]
Closable feature can be ported from PrimeReact.

Thank you!

**Comment by saenzramiro** [comment]
Any progress on this?
